doc/user_guide/interactions.rst

```diff
@@ -569,69 +540,107 @@ With point selections, an input element can be added to the chart to establish a
         y='Miles_per_Gallon:Q',
         # We need to set a constant domain to preserve the colors
         # when only one region is shown at a time
         color=alt.Color('Origin:N', scale=alt.Scale(domain=options)),
-        tooltip='Name:N'
     ).add_params(
         selection
     ).transform_filter(
         selection
     )
 
-Logic-Driven Input Element Binding
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-So far we have seen the use of selections to match values in our data.
-When using checkbox bindings,
-we want to instead use the state of the checkbox as a True/False condition
-and execute some action depending on whether it is checked or not.
-When we are using a checkbox as a toggle like this,
-we need to use `param` instead of `selection_point`,
-since we don't want to check if there are True/False values in our data,
-just if the value of the check box is True (checked) or False (unchecked):
+In addition to the widgets listed in the table above,
+Altair has access to `any html widget <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input>`_
+via the more general ``binding`` function.
+In the example below,
+we use a search input to filter points that match the search string exactly.
+You can hover over the points to see the car names
+and try typing one into the search box, e.g. ``vw pickup``
+to see the point highlighted.
 
 .. altair-plot::
 
-    bind_checkbox = alt.binding_checkbox(name='Scale point size by "Acceleration": ')
-    param_checkbox = alt.param(bind=bind_checkbox)
-
-    alt.Chart(cars).mark_point().encode(
+    search_input = alt.selection_point(
+        value='',
+        fields=['Name'],
+        bind=alt.binding(
+            input='search',
+            placeholder="Car model",
+            name='Search ',
+        )
+    )
+    alt.Chart(data.cars.url).mark_point(size=60).encode(
         x='Horsepower:Q',
         y='Miles_per_Gallon:Q',
-        size=alt.condition(
-            param_checkbox,
-            alt.Size('Acceleration:Q'),
-            alt.value(25)
+        tooltip='Name:N',
+        opacity=alt.condition(
+            search_input,
+            alt.value(0.8),
+            alt.value(0.1)
         )
     ).add_params(
-        param_checkbox
+        search_input
     )
 
-Similarly, if we want to create a condition
-where we use the value of a slider
-we can use `param` like so:
+It is not always useful to require an exact match to the search syntax,
+and when we will be learning about :ref:`expressions`,
+we will see how we can match partial strings via a regex instead.
+
+Data-Driven Comparisons
+-----------------------
+
+So far we have seen the use of selections
+to lookup points with precisely matching values in our data.
+This is often useful,
+but sometimes we might want to make a more complex comparison
+than an exact match.
+For example,
+we might want to create a condition
+where we want to access the value of a slider
+to use directly in a comparison with values of the data.
```

Comment:
```
For example,
we might want to create a condition
where we want to access the value of a slider
to use directly in a comparison with values of the data.
```
It is a bit hard for me to understand this sentence, especially the last part. After reading it two times I get what you mean. Maybe split in two sentences?